lib/wings/services/file_converter_service.rb

```diff
@@ -1,67 +1,18 @@
 # frozen_string_literal: true
 # Conversion service for going between files on a valkyrie resource and files on an active fedora object
 module Wings
+  # @deprecated use generic Wings converters instead.
+  # @see ModelTransformer, ActiveFedoraConverter
```

Comment:
all this behavior is now delegated to the generic wings converter/transformer pair, and this class is deprecated.